CHANGES.rst

```diff
@@ -23,6 +22,7 @@ Unreleased
 -   Deprecate ``SQLALCHEMY_COMMIT_ON_TEARDOWN`` as it can cause various
     design issues that are difficult to debug. Call
     ``db.session.commit()`` directly instead. :issue:`216`
+-   Fix session get_bind signature for SQLAlchemy 1.4. :issue:`953`
```

Comment:
Use Sphinx syntax:

```
``get_bind``
```

Reply:
Actually, this can be a new section 2.5.2, it's just a bug fix against a bug fix in 2.5.0.